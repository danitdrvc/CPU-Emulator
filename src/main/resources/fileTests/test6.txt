CMP QWORD R0, R1
ADD QWORD R0, R1
SUB QWORD R3, R1
HALT