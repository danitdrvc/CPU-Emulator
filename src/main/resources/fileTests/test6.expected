[15, 5, 0, -4]